plugins/inputs/sqlserver/README.md

```diff
@@ -66,20 +66,25 @@ GO
 
   # database_type = "AzureSQLDB"
 
-  ## A list of queries to include. If not specified, all the above listed queries are used.
+  ## A list of queries to include. 
   # include_query = []
 
   ## A list of queries to explicitly ignore.
-  # exclude_query = []
+  ## Query Store queries (AzureSQLDBQueryStoreRuntimeStatistics and AzureSQLDBQueryStoreWaitStatistics) are excluded by default.
+  ## If you enable Query Store queries, the collection interval should comply with the following restriction: 15m =< collection interval <= 2h. 
+  ## Intervals shorter than 15m may cause higher performance impact on source. Intervals longer than 2h may cause some Query Store data not to be collected.
+  # exclude_query = ["AzureSQLDBQueryStoreRuntimeStatistics", "AzureSQLDBQueryStoreWaitStatistics"]
 
   ## Queries enabled by default for database_type = "AzureSQLManagedInstance" are - 
   ## AzureSQLMIResourceStats, AzureSQLMIResourceGovernance, AzureSQLMIDatabaseIO, AzureSQLMIServerProperties, AzureSQLMIOsWaitstats, 
   ## AzureSQLMIMemoryClerks, AzureSQLMIPerformanceCounters, AzureSQLMIRequests, AzureSQLMISchedulers
 
   # database_type = "AzureSQLManagedInstance"
 
+  ## A list of queries to include.
   # include_query = []
 
+  ## A list of queries to explicitly ignore.
   # exclude_query = []
```

Comment:
Somehow `database_type`, `include_query` and `exclude_query` entries are duplicated. Can you please cleanup those?

Reply:
@srebhan, they are duplicated intentionally. **README** and **sampleConfig** contain different **include_query** and **exclude_query** per each **database_type**. Do you think this is wrong?

Reply:
Ok fine with me.